client/src/app/site/pages/meetings/modules/poll/components/base-poll-form/base-poll-form.component.ts

```diff
@@ -300,9 +304,7 @@ export abstract class BasePollFormComponent extends BaseComponent implements OnI
     public showMinMaxVotes(data: any): boolean {
         const selectedPollMethod: FormPollMethod = this.pollMethodControl.value;
         return (
-            (selectedPollMethod === `Y` ||
-                selectedPollMethod === `N` ||
-                (selectedPollMethod === `YNA` && this.allowToSetMinMaxOnYNA)) &&
+            (selectedPollMethod === `Y` || (selectedPollMethod !== `yna` && this.allowToSetMinMax)) &&
```

Comment:
```suggestion
            (selectedPollMethod === `Y` || (selectedPollMethod !== `YNA` && this.allowToSetMinMax)) &&
```

Should be uppercase ig?

Reply:
no yna in lower case is the list option

Reply:
Please at least use the enum from `FormPollMethod` to prevent further confusion in the future.